services/nvidia-gpu-operator/23.3.2/helmrelease/extra-images.txt

```diff
@@ -0,0 +1,9 @@
+nvcr.io/nvidia/gpu-feature-discovery:{{ .Values.gfd.version }}
+nvcr.io/nvidia/k8s/container-toolkit:{{ regexReplaceAllLiteral "-.+$" .Values.toolkit.version "" }}-centos7
+nvcr.io/nvidia/k8s/container-toolkit:{{ regexReplaceAllLiteral "-.+$" .Values.toolkit.version "" }}-ubuntu20.04
+nvcr.io/nvidia/k8s/container-toolkit:{{ regexReplaceAllLiteral "-.+$" .Values.toolkit.version "" }}-ubi8
+nvcr.io/nvidia/cloud-native/gpu-operator-validator:{{ .Values.validator.version }}
+nvcr.io/nvidia/cloud-native/dcgm:{{ .Values.dcgm.version }}
+nvcr.io/nvidia/k8s/dcgm-exporter:{{ .Values.dcgmExporter.version }}
+nvcr.io/nvidia/k8s-device-plugin:{{ .Values.devicePlugin.version }}
+nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda10.2
```

Comment:
i think we can remove this [automation](https://github.com/mesosphere/kommander-applications/blob/main/.github/workflows/kommander-branch.yaml#L110-L124) now that these images are defined here. 

maybe just a follow up to remove the [script](https://github.com/mesosphere/kommander-applications/blob/main/hack/update-nvidia.sh)/job

cc @faiq since you set all that up

Reply:
Can someone from your team follow up on this?